### PR TITLE
1.10 discussion correct

### PR DESCRIPTION
**Pull request by alaninmcr**
Stuart can you look at this?

### CODE REVIEW

**Review by stuzart — COMMENTED** [review]
some comments to the changes

File: app/models/assay.rb [hunk 1]
```diff
@@ -65,6 +65,8 @@ class Assay < ApplicationRecord
 
   enforce_authorization_on_association :study, :view
 
+  acts_as_discussable
```
[the remark this answers is not in the document]
Comment (alaninmcr) [reply app/models/assay.rb:68]: Done

File: app/views/institutions/show.html.erb [hunk 2]
```diff
@@ -21,6 +21,13 @@
 
   <div class="col-md-3 col-sm-4">
     <%= render :partial => "layouts/contribution_section_box_avatar", :locals => { :object => @institution } -%>
+
+    <% if @institution.respond_to?(:discussion_links) -%>
```
[the remark this answers is not in the document]
Comment (alaninmcr) [reply app/views/institutions/show.html.erb:25]: Done

File: test/fixtures/files/json/content_compare/max_assay.json [hunk 3]
```diff
@@ -24,7 +24,8 @@
       "technology_type": {
         "label": null,
         "uri": null
-      }
+      },
+      "discussion_links" : null
```
[the remark this answers is not in the document]
Comment (alaninmcr) [reply test/fixtures/files/json/content_compare/max_assay.json:28]: Added to all

File: app/serializers/base_serializer.rb [hunk 4]
```diff
@@ -6,6 +6,13 @@ class BaseSerializer < SimpleBaseSerializer
 
   attribute :policy, if: :show_policy?
 
+  attribute :discussion_links,  if: -> { object.respond_to?(:discussion_links) } do
+    object.discussion_links.collect do |link|
+      {id:link.id, label: link.label, url: link.url}
+    end
+  end
+
+
```
[the remark this answers is not in the document]
Comment (alaninmcr) [reply app/serializers/base_serializer.rb:15]: Removed from ContributedResourceSerializer

File: app/models/programme.rb [hunk 5]
```diff
@@ -54,6 +54,8 @@ class Programme < ApplicationRecord
       joins: [:funding_codes_as_text]
   )
 
+  acts_as_discussable
```
[the remark this answers is not in the document]
Comment (alaninmcr) [reply app/models/programme.rb:57]: Done

File: app/models/assay.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (alaninmcr) [reply app/models/assay.rb:68]: Done

File: app/views/institutions/show.html.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (alaninmcr) [reply app/views/institutions/show.html.erb:25]: resource main content right is not included in all resources e.g. not in project

**Pull request merged**